mobile/lib/src/application/settings/settings_cubit.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:polydodo/src/domain/settings/settings.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+part 'settings_state.dart';
+
+class SettingsCubit extends Cubit<SettingsState> {
+  SettingsCubit() : super(SettingsLoadInProgress()) {
+    getSettings();
+  }
```

Comment:
```suggestion
class SettingsCubit extends Cubit<SettingsState> {
  SharedPreferences prefs;

  SettingsCubit() : super(SettingsLoadInProgress()) {
    getSettings();
  }
```

En enregistrant SharedPreference, tu pourra t'éviter de toujours de faire await SharedPreference.getInstance()

Reply:
J'ai un vague souvenir que y'avait une raison pourquoi j'ai fait ça.. :O mais honnêtement j'vois plus pourquoi alors ok.